mendeleydesktop.appdata.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess I thought about including this kind of file before and it makes sense, but my last attempt of bringing this package to rpmfusion took 2 years of review, and by that time I gave up.

Do you really want to be held to updating this file when it is never used?

Reply:
I read about your attempt and thank for setting a blueprint. I look at the spec file and clean up a bit to get as close to Fedora packaging guideline as you noticed. The fixes are set and I am about to upload files on the forke page.